### ISSUE TITLE
Make better choice for Grandma number

### ISSUE THREAD

**Issue by zsc4** [staticvariablejames/ChooseYourOwnLump#6]
I think this is a really great mod, and it has already helped me a lot.  I have just one suggestion.  Currently, the required number of Grandmas shows possibilities only going up to 600, a number that is hardwired into the code.  This can make it a bit of a hassle to use this mod if you have a lot more than 600 grandmas, especially since they play such a central role in the game.  I replaced these instances of 600 with the expression **Game.Objects['Grandma'].amount**, and the mod now displays options starting with your current number of grandmas and works its way down.  I would highly recommend that you adopt this change as well.

The four places where this needs to be changed are lines 194, 222, 343, and 546.

**Comment by alainbryden** [comment]
Old issue, but for future readers I'm pretty sure the reason for this decision is that the effect grandmas have on sugar lump maturation time is capped at 600 in-game, so any number of grandmas greater than 600 has the same effect as 600, it's a waste of time to test those scenarios.

If the game finds that 600 is a viable number, then you are free to leave grandmas untouched.

**Comment by staticvariablejames** [comment]
Correct. The heavenly upgrade "Sugar aging process" is the one which allows grandmas to affect sugar lump maturation times, and that heavenly upgrade is capped at 600 grandmas.
